UPGRADE_GUIDE.md

```diff
@@ -5,6 +5,107 @@ To see discussion around these API changes, please refer to the
 [changelog](/CHANGELOG.md) and visit the commits and issues they
 reference.
 
+0.13.x -> 1.0.0
+----------------
+
+- React `0.14.x` is now required
```

Comment:
I'm running React@0.13.3 with React-Router@1.0.0-beta3, so it's not a requirement!

And their package.json still uses react-0.13 in dev https://github.com/rackt/react-router/blob/master/package.json#L51